bentoml/yatai/yatai_service.py

```diff
@@ -40,13 +40,20 @@ def get_yatai_service(
         scheme, addr = parse_grpc_url(channel_address)
 
         if scheme in ('grpcs', 'https'):
-            client_cacert_path = (
-                config().get('yatai_service', 'client_certificate_file')
+            tls_root_ca_cert = (
+                config().get('yatai_service', 'tls_root_ca_cert')
+                or config().get('yatai_service', 'client_certificate_file')
```

Comment:
👍 add an inline comment explaining this config key is meant for backward compatibility purposes?

Reply:
Yes, makes sense. Sometimes that's faster than `git blame`. I added a short comment.

Reply:
I haven't seen that you are using a wrapper for the ConfigParser which checks if the section, key pair is available in the configuration file. Should I add also the `client_certificate_file` to the default configuration? Do you have a process for changing the naming of configuration options?

Reply:
@jloehel great question, we don't have a process for changing the configuration options yet, although the configparser wrapper tries to read from the config file and env var as a simple hierarchy config system, although it's quite limited. We are considering switching to use an existing framework like https://github.com/adobe/himl